packages/manager/modules/pci/src/projects/project/inactive/inactive.routing.js

```diff
@@ -0,0 +1,24 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.inactive', {
+    url: '/inactive',
+    views: {
+      modal: {
+        component: 'pciProjectInactive',
+      },
+    },
+    layout: 'modal',
+    params: {
+      billingStatus: null,
+      projectStatus: null,
+    },
+    resolve: {
+      billingStatus: /* @ngInject */ ($transition$) =>
+        $transition$.params().billingStatus,
+      goBack: /* @ngInject */ ($window) => () => $window.history.back(),
```

Comment:
it's better to provide a state with ```state.go(...)``` otherwise if i access the page directly by typing the url the back button won't bring me to the section

Reply:
This is fixed. Thanks :)